### PR TITLE
Adjust to Radicle contracts

### PR DESCRIPTION
**Pull request by xla**
Signed-off-by: Alexander Simmerl <a.simmerl@gmail.com>

### CODE REVIEW

**Review by CodeSandwich — APPROVED** [review]
Very cool!

File: config/ropsten.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "network": "ropsten",
+  "startBlock": 9655941,
+  "governorAddress": "0x559d6F96945072fC567f7e03167364eBf5C76A52",
+  "tokenAddress": "0x221577dB84e6eA22ed18dDCC14298EA45d894644"
+}
```
Comment (cloudhead) [inline config/ropsten.json:6]: These aren't correct, will have to check.

File: config/homestead.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "network": "mainnet",
+  "startBlock": 11863739,
+  "governorAddress": "0x690e775361AD66D1c4A25d89da9fCd639F5198eD",
+  "tokenAddress": "0x31c8EAcBFFdD875c74b94b077895Bd78CF1E64A3"
+}
```
Comment (cloudhead) [inline config/homestead.json:6]: These are correct.

File: config/ropsten.json (hunk 1 above)
Comment (xla) [reply config/ropsten.json:6]: Used the ones from the initial deploy, we can adjust if you have updated ones or move to rinkeby.

**Pull request closed without merging**